README.md

```diff
@@ -1,25 +1,29 @@
 Overview
 ==
-As the popularity of the library [minifb](https://crates.io/crates/minifb) shows, it is useful to put a 2D buffer/image
-on a window in a platform-independent way. Minifb's approach to doing window management itself, however, is problematic
+
+Enables software rendering via drawing an image straight to a window.
+
+Softbuffer integrates with the [raw-window-handle](https://crates.io/crates/raw-window-handle) crate
+to allow writing to a window in a cross-platform way while using the very high quality dedicated window management
```

Comment:
I find "writing to a window" quite vague. Can we clarify this by adding "pixels" or "colors" or something?